docs/core/tools/dotnet-tool-list.md

```diff
@@ -0,0 +1,59 @@
+---
+title: dotnet tool list command - .NET Core CLI
+description: The dotnet tool list command lists the specified .NET Core Global Tool from your machine.
+author: mairaw
+ms.author: mairaw
+ms.date: 04/25/2018
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.workload: 
+  - dotnetcore
+---
+# dotnet tool list
+
+[!INCLUDE [topic-appliesto-net-core-2-1plus.md](../../../includes/topic-appliesto-net-core-2-1plus.md)]
+
+## Name
+
+`dotnet tool list` - Lists all [.NET Core Global Tools](global-tools.md) currently installed on your machine or specified path.
```

Comment:
Current text could be read: All global tools installed anywhere on your machine. 

"currently installed in the default directory on your machine or specified path.